@here/harp-geoutils/lib/projection/MercatorProjection.ts

```diff
@@ -64,28 +64,33 @@ export class MercatorProjection implements Projection {
     }
 
     projectPoint<WorldCoordinates extends Vector3Like>(
-        geoPoint: GeoCoordinatesLike,
-        result?: WorldCoordinates
+        geoPointLike: GeoCoordinatesLike,
+        result?: WorldCoordinates,
+        normalize: boolean = true
     ): WorldCoordinates {
-        let normalized: GeoCoordinates;
+        let geoPoint: GeoCoordinates;
 
-        if (geoPoint instanceof GeoCoordinates) {
-            normalized = geoPoint.normalized();
+        if (geoPointLike instanceof GeoCoordinates) {
```

Comment:
How about:
normalize = !(normalize === false);

Reply:
looks like Hax, but I guess it is a standard way of converting undefined & false to false?

Reply:
it is shorter, that is all. There are so many uses of it if parameters can be undefined

Reply:
all good